winpr/libwinpr/input/keycode.c

```diff
@@ -623,6 +902,17 @@ DWORD GetKeycodeFromVirtualKeyCode(DWORD vkcode, DWORD dwFlags)
 			}
 		}
 	}
+	else if (dwFlags & KEYCODE_TYPE_XKB)
+	{
+		for (index = 0; index < 256; index++)
```

Comment:
Maybe use `ARRAYSIZE(KEYCODE_TO_VKCODE_XKB)` here?

Reply:
Yep, did also the same for `KEYCODE_TO_VKCODE_APPLE`, and `KEYCODE_TO_VKCODE_EVDEV`.